tests/ui/thread-local/thread-local-static.rs

```diff
@@ -12,7 +12,6 @@ const fn g(x: &mut [u32; 8]) {
     //~^^ ERROR thread-local statics cannot be accessed
     //~| ERROR mutable references are not allowed
     //~| ERROR use of mutable static is unsafe
-    //~| referencing statics
```

Comment:
Here this was not a literal duplicate -- the other error is "thread-local statics cannot be accessed". So the old error was even wrong, since STATIC_VAR_2 is a thread-local static, not a regular static, and saying that this is "unstable" and needs the const_refs_to_static feature gate is not correct.